angular/src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,42 @@
+div.container.valign-wrapper {
+    width: 100%;
+    height: 50px;
+    display: flex;
```

Comment:
Be careful, `flex-flow` defaults to `row nowrap` unless you use a CSS library that changes it. Which means your component's layout will be broken beyond recognition if the viewport is too narrow for all the elements to be displayed as you want them to.